qa/os/src/test/java/org/elasticsearch/packaging/util/Shell.java

```diff
@@ -68,6 +76,20 @@ public Result runIgnoreExitCode(String script) {
         return runScriptIgnoreExitCode(getScriptCommand(script));
     }
 
+    public void chown(Path path) throws Exception {
+        Platforms.onLinux(() -> run("chown -R elasticsearch:elasticsearch " + path));
+        Platforms.onWindows(() -> run(
+            "$account = New-Object System.Security.Principal.NTAccount '" + System.getenv("username")  + "'; " +
```

Comment:
Don't we have the constant ARCHIVE_USER for use here and above?

Reply:
There's `ARCHIVE_OWNER` but that's part of the `ArchiveTests` and has hard coded values.